tests/kubeUtils.sh

```diff
@@ -35,9 +35,13 @@ function create_namespace(){
 
 # Bring up control plane
 function deploy_istio() {
-    local istio_install="${1}"
     print_block_echo "Deploying ISTIO"
-    ${K8CLI} -n ${NAMESPACE} create -f "${istio_install}" \
+    # order is important
+    ${K8CLI} -n ${NAMESPACE} create -f $ROOT/kubernetes/istio-install/istio-manager.yaml \
```

Comment:
Why not using one command for all. If you want to keep that way, then change "Failed to create control plane".